src/main/java/mate/academy/dao/MovieSessionDao.java

```diff
@@ -6,9 +6,13 @@
 import mate.academy.model.MovieSession;
 
 public interface MovieSessionDao {
-    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+    MovieSession add(MovieSession movieSession);
 
     Optional<MovieSession> get(Long id);
 
-    MovieSession add(MovieSession session);
+    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+
+    boolean update(MovieSession movieSession);
```

Comment:
Consider changing the return type of the `update` method to `MovieSession` or `void` to better reflect its operation. Returning a boolean might not provide enough information about the update process.